tests/requirements/test_tasklet_priority.rs

```diff
@@ -0,0 +1,26 @@
+use assert_cmd::Command;
+use test_binary::build_test_binary;
+
+/// @SRS{ROS-FUN-RTOS-10}
+/// @SRS{ROS-FUN-RTOS-90}
+/// @SRS{ROS-FUN-RTOS-100}
+/// @SRS{ROS-FUN-RTOS-110}
+#[cfg_attr(not(doc), test)]
+fn req_test_tasklet_priority() {
+    let test_bin_path =
+        build_test_binary("test-tasklet-priority", "testbins").expect("error building test binary");
+
+    Command::new(test_bin_path)
+        .timeout(std::time::Duration::from_secs(1))
+        .assert()
+        .success()
+        .code(0)
+        .stdout(
+            r#"TaskB: 1
+TaskB: 2
+TaskB: 3
+TaskB: 4
+TaskB: 5
+"#,
```

Comment:
move quit to TaskC and check if it executes after TaskB finishes?

Reply:
Added a separate test for this.